ingenialink/ethercat/servo.py

```diff
@@ -320,6 +323,59 @@ def set_pdo_watchdog_time(self, timeout: float) -> None:
             self.ETHERCAT_PDO_WATCHDOG, self.SECONDS_TO_MS_CONVERSION_FACTOR * timeout
         )
 
+    def read_eeprom(
```

Comment:
Users should not be using this and the name can lead to confusions.
The first thing a user will think of an eeprom of the drive is the eeprom of config.

I would call this read_esc_eeprom.